code/modules/events/aurora_caelus.dm

```diff
@@ -37,13 +37,13 @@
 		if(istype(affected_area, /area/station/service/kitchen))
 			for(var/turf/open/kitchen in affected_area)
 				kitchen.set_light(1, 0.75)
-			if(!prob(1) && !SSevents.holidays?[APRIL_FOOLS])
-				continue
+//			if(!prob(1) && !SSevents.holidays?[APRIL_FOOLS]) // SKYRAT EDIT BEGIN - Always in your kitchen
+//				continue
```

Comment:
While both skyrat edits ARE in the same proc, they're technically different ones. This one should be //SKYRAT EDIT REMOVAL